cartography/intel/aws/rds.py

```diff
@@ -106,16 +106,96 @@ def load_rds_instances(neo4j_session, data, region, current_aws_account_id, aws_
             aws_update_tag=aws_update_tag
         )
         _attach_ec2_security_groups(neo4j_session, rds, aws_update_tag)
+        _attach_ec2_subnet_groups(neo4j_session, rds, region, current_aws_account_id, aws_update_tag)
     _attach_read_replicas(neo4j_session, read_replicas, aws_update_tag)
 
 
+def _attach_ec2_subnet_groups(neo4j_session, instance, region, current_aws_account_id, aws_update_tag):
+    """
+    Attach RDS instance to its EC2 subnets
+    """
+    attach_rds_to_subnet_group = """
+    MERGE(sng:DBSubnetGroup{id:{sng_arn}})
+    ON CREATE SET sng.firstseen = timestamp()
+    SET sng.name = {DBSubnetGroupName},
+    sng.vpc_id = {VpcId},
+    sng.description = {DBSubnetGroupDescription},
+    sng.status = {DBSubnetGroupStatus},
+    sng.lastupdated = {aws_update_tag}
+    WITH sng
+    MATCH(rds:RDSInstance{id:{DBInstanceArn}})
+    MERGE(rds)-[r:MEMBER_OF_DB_SUBNET_GROUP]->(sng)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    """
+    if not instance.get('DBInstanceArn'):
+        logger.debug("Expected RDSInstance to have a DBInstanceArn but it doesn't.  Here is the object: %r", instance)
+    elif not instance.get('DBSubnetGroup'):
+        logger.debug("Expected RDSInstance to have a DBSubnetGroup but it doesn't.  Here is the object: %r", instance)
+    else:
+        db_sng = instance.get('DBSubnetGroup')
+        arn = _get_db_subnet_group_arn(region, current_aws_account_id, db_sng.get('DBSubnetGroupName'))
+        neo4j_session.run(
+            attach_rds_to_subnet_group,
+            sng_arn=arn,
+            DBSubnetGroupName=db_sng.get('DBSubnetGroupName'),
+            VpcId=db_sng.get("VpcId", None),
```

Comment:
dict.get's default return value is always None so there's no need to specify it here (and in the calls below)